File: 66-plus-one/66-plus-one.py
class Solution(object):
    def plusOne(self, d):
        for i in range(len(d)):
            d[i] = str(d[i])
        
        d=''.join(d)
        d=int(d)+1
        d=str(d)
        
        a=[]
        for i in d:
            a.append(int(i))
            
        return a


        

File: 66-plus-one/66-plus-one.java
class Solution {
    public int[] plusOne(int[] a) {
        int n=a.length;
        if (a[n-1]!=9){
            a[n-1]+=1;
            return a;
        }
        for(int i=n-1;i>=0;--i){
            if(a[i]==9){
                a[i]=0;
            }
            else{
                a[i]+=1;
                return a;
            }
        }
        
        int []ex=new int[a.length+1];
        ex[0]=1;
        
        return ex;
    }
}